### PR TITLE
updates

### PR DESCRIPTION
**Pull request by sateeshperi**
@netlify /blog/2024/maintainers-minutes-2024-11-29

### CODE REVIEW

**Comment by sateeshperi** [comment]
@nf-core-bot fix linting

**Comment by sateeshperi** [comment]
@nf-core-bot fix linting

File: sites/main-site/src/content/blog/2024/maintainers-minutes-2024-11-29.mdx [hunk 1]
```diff
@@ -22,80 +23,109 @@ by providing brief summaries of the monthly team meetings.
...
--   [Stricter SemVer versioning](stricter-semver-versioning)
--   [Push for BAM to CRAM?](push-for-bam-to-cram)
--   [Removal of `workflowCitation()`](removal-of-workflowcitation)
--   [Recent CI approach changes](recent-ci-approach-changes)
--   [Modules piling up](modules-piling-up)
+-   [Stricter SemVer versioning](#Stricter SemVer versioning)
+-   [Push for BAM to CRAM?](#Push for BAM to CRAM?)
+-   [Removal of `workflowCitation()`](#Removal of `workflowCitation()`)
+-   [Recent CI approach changes](#Recent CI approach changes)
+-   [Modules piling up](#Modules piling up)
```
Comment (mashehu) [inline sites/main-site/src/content/blog/2024/maintainers-minutes-2024-11-29.mdx:30]: Nope these were links with the heading ids (which are automatically generated), I don't think your changes are still linking correctly

File: sites/main-site/src/content/blog/2024/maintainers-minutes-2024-11-29.mdx [hunk 2]
```diff
@@ -22,80 +23,109 @@ by providing brief summaries of the monthly team meetings.
...
+a release.
 
-We agreed that Jon would prepare a [PR](https://github.com/nf-core/website/pull/2842) where the rest of the community can propose what they consider to be the 'rules' about what defines different types of release, following SemVer as far as possible (note the PR has since been merged, but it's still open for updating by the rest of the community).
-Then with these specifications <Profile username="mirpedrol">Júlia Mir Pedrol</Profile> and the infrastructure team would explore tooling to propose during linting the 'best fitting' version bump prior to a new release.
+The group agreed that Jon would prepare a [PR](https://github.com/nf-core/website/pull/2842) where the community could collectively define ‘rules’ for different release types, adhering to SemVer principles as much as possible. (Note: The PR has since been merged, but it remains open for updates from the community.) Following this, Júlia Mir Pedrol and the infrastructure team committed to exploring tooling that could suggest the most appropriate version bump during the linting process before a new release.
```
Comment (jfy133) [inline sites/main-site/src/content/blog/2024/maintainers-minutes-2024-11-29.mdx:51]: Missing the badge thing for Júlia

File: sites/main-site/src/content/blog/2024/maintainers-minutes-2024-11-29.mdx [hunk 3]
```diff
@@ -22,80 +23,109 @@ by providing brief summaries of the monthly team meetings.
...
+
+Pipeline CI Workflow Updates:
+
+Pipelines using GPU-dependent modules must update their CI workflows to support GPU testing. Currently, nf-core/methylseq serves as a test case for these CI updates.
 
-The two main changes were the use of nf-test 'shards', allowing different tests defined in the nf-test script file to run in parallel rather than sequentially.
-This means a faster development cycle of modules by having faster successes, or more importantly, fails.
-A second major change is the ability to properly test modules that use tools that require GPUs, with a specific tag now triggering the test to run on 'self-hosted' runners outside of GitHub using nf-core credits kindly donated by AWS.
+The updated CI workflow involves two main github actions:
+• **nf-test-shard**: Performs a dry run of nf-test with the `--changed-since` flag, filtering by tags. It outputs the test shard and total number of shards.
```
Comment (jfy133) [inline sites/main-site/src/content/blog/2024/maintainers-minutes-2024-11-29.mdx:105]: Please use markdown bullets

**Comment by sateeshperi** [comment]
@nf-core-bot fix linting

**Comment by mashehu** [comment]
@nf-core-bot fix linting

File: sites/main-site/src/content/blog/2024/maintainers-minutes-2024-11-29.mdx [hunk 4]
```diff
@@ -22,80 +23,109 @@ by providing brief summaries of the monthly team meetings.
...
 
-Jon brought up that different pipelines take different approaches, and few follow 'true' SemVer rules.
-The main issue was 'major releases': in SemVer, going from 1.0.0 to 2.0.0 indicates a _breaking change_, where the way a user (or computer) interacts with the software has changed in a way that the way they are used to using the software will not work any more.
-However in many nf-core pipelines, developers have used the major version bump to indicate major new functionalities or large back-end refactoring of the code base, even if it doesn't affect the way a user interacts with the pipeline.
+Jon pointed out that different pipelines take different approaches, with few adhering strictly to ‘true’ SemVer rules. The primary issue revolved around ‘major releases.’ According to SemVer, transitioning from version 1.0.0 to 2.0.0 signifies a breaking change—a change that alters the software’s functionality such that existing usage patterns no longer work. However, many nf-core pipelines have used major version bumps to indicate substantial new functionalities or extensive back-end codebase refactoring, even when these changes do not disrupt user interactions.
```
[the remark this answers is not in the document]
Comment (sateeshperi) [reply sites/main-site/src/content/blog/2024/maintainers-minutes-2024-11-29.mdx]: how about now ?

**Review by jfy133 — APPROVED** [review]

**Pull request merged**